style.css

```diff
@@ -1,81 +1,73 @@
-
-body{
-    background: rgb(4,0,66);
-    background: radial-gradient(circle, rgba(4,0,66,1) 0%, rgba(22,24,62,1) 35%, rgba(0,114,138,1) 100%);
+body {
+    background: rgb(4, 0, 66);
+    background: radial-gradient(circle, rgba(4, 0, 66, 1) 0%, rgba(22, 24, 62, 1) 35%, rgba(0, 114, 138, 1) 100%);
     margin: 0;
     padding: 0;
 }
 
-#donate, #login{
+#donate, #login {
     float: right;
     width: 175px;
     max-width: 100%;
     height: 50px;
     background-color: white;
     border: none;
     border-radius: 20px;
-    text-size: 5px;
+    font-size: 5px;
     opacity: unset;
 }
 
-.header{
+.header {
     position: sticky;
     top: 0;
     display: flex;
-    justify-content: flex-end; /* Align buttons to the right */
-    gap: 20px; /* Add space between buttons */
+    justify-content: flex-end;
+    gap: 20px;
     background-color: rgba(0, 0, 0, 0.5);
 }
 
-
 .main {
     display: flex;
     flex-direction: column;
 }
 
-.landing{
+.landing {
     background-color: #0c0e33;
     width: 100%;
 }
 
-.title{
+.title {
     text-align: center;
 }
 
-.FAQ{
+.FAQ {
     background-color: #3c240e;
     width: 100%;
-    
 }
 
-#body{
+#body {
     margin: 0;
 }
 
-.sponsor_divs {
-    margin: 20px auto;
-    max-width: 1200px;
-    padding: 20px;
-}
-
-.button-container {
-    display: flex;
-    justify-content: center;
-    margin-top: 20px;
-    gap: 20px;
+.question {
+    background-color: #3c240e;
+    padding: 10px;
+    cursor: pointer;
+    color: white;
 }
 
-.large-button {
```

Comment:
you've deleted a large chunk of css which was being used by sponsorships... Why?